bin/prepare-release.sh

```diff
@@ -102,21 +102,13 @@ run $SED -i.bak -E \
     "${WORKSPACE_ROOT}/${FILE}"
 run rm "${WORKSPACE_ROOT}/${FILE}.bak"
 
-### GLEAN-PREVIEW ###
+### GLEAN RLB ###
 
 # Update the version of the glean-core dependency
 
-FILE=glean-core/preview/Cargo.toml
```

Comment:
glean-preview was versioned independently of glean-core (it's at 0.0.5).
The code previously here ensured we update the glean-core version it depends on.

This needs changes to update both the glean-core version it depends on and its own version.

Reply:
Yes, this is what the file is doing now: i tested it locally and it updates both the glean-core version and its own version.

Reply:
uh fancy, I can now see why. it's both a line with just `version = ...`